packages/valory/skills/termination_abci/behaviours.py

```diff
@@ -156,7 +179,7 @@ def check_for_signal(self) -> Generator[None, None, bool]:
     def _get_latest_removed_owner_event(self) -> Generator[None, None, Optional[Dict]]:
         """Returns the latest event in which the service owner was removed from the set of owners of the safe."""
         response = yield from self.get_contract_api_response(
-            performative=ContractApiMessage.Performative.GET_STATE,
+            performative=ContractApiMessage.Performative.GET_STATE,  # type: ignore
```

Comment:
any idea why mypy always complains about this? :shrug: 
`assert isinstance(ContractApiMessage.Performative.GET_STATE, ContractApiMessage.Performative)`

Reply:
I gave up after a while. It passes on another (private) repo.